index.html

```diff
@@ -24,19 +24,30 @@
 <body>
    <section class="body">
     <section class="sect">
+
+      
+      <div id="drop-menu" class="menu">
+        <img onclick="myFunction1()" src="images/Iconx.png" alt="cancel" class="cancel">
+        <ul>
+          <li><a href="#Portfolio" onclick="myFunction1()">Portfolio</a><br><hr></li>
+          <li><a href="#About" onclick="myFunction1()">About</a><br><hr></li>
+          <li><a href="#Contacts" onclick="myFunction1()">Contacts</a><br><hr></li>
+        </ul>
+      </div>
```

Comment:
- [ ] <b>REQUIRED:</b> When I click on hamburger menu button, nothing is happening. kindly fix it.

Reply:
There seems to be no problem here, so I think the issue may be from your side. Please kindly double check to see.

Reply:
This issue is well resolved! :heavy_checkmark